internal/plugins/ansible/v1/scaffolds/internal/templates/molecule/mdefault/molecule.go

```diff
@@ -63,7 +63,7 @@ provisioner:
         config_dir: ${MOLECULE_PROJECT_DIRECTORY}/config
         samples_dir: ${MOLECULE_PROJECT_DIRECTORY}/config/samples
         operator_image: ${OPERATOR_IMAGE:-""}
-        operator_pull_policy: ${OPERATOR_PULL_POLICY:-"Always"}
+        pull_policy: ${OPERATOR_PULL_POLICY:-"Always"}
         kustomize: ${KUSTOMIZE_PATH:-kustomize}
```

Comment:
@asmacdo you will need to change here as well.  https://github.com/operator-framework/operator-sdk/pull/3654/files :-)